src/gui/tray/Window.qml

```diff
@@ -391,7 +391,8 @@ Window {
                     Menu {
                         id: appsMenu
                         y: (trayWindowAppsButton.y + trayWindowAppsButton.height + 2)
-                        width: Math.min(contentItem.childrenRect.width + 4, Style.trayWindowWidth / 2)
+                        readonly property Item listContentItem: contentItem.contentItem
+                        width: Math.min(listContentItem.childrenRect.width + 4, Style.trayWindowWidth / 2)

```

Comment:
What's the benefit of declaring a new property instead of only accessing `contentItem.contentItem.childrenRect.width` directly? Asking for a ~friend~ noise-to-signal ratio.

Reply:
Apart from that, it's an insanely simple as well as insanely insane solution :smiley:

Reply:
The new property is mostly there for the name, if there's several indirections like that you loose the intent of the author... it can go wrong really fast in QML, I've seen enough "parent.parent.parent" to fill a book for instance... "contentItem.contentItem" makes the next developer wonder "why not contentItem" for instance, at least trying to express in this imperfect language that I expect a content item from a list within that menu... Yeah, I know...